modules/RTC/ortc/RTCPeerConnection.js

```diff
@@ -0,0 +1,532 @@
+/* global __filename, RTCIceGatherer, RTCIceTransport */
+
+import { getLogger } from 'jitsi-meet-logger';
+import yaeti from 'yaeti';
+import { InvalidStateError } from './errors';
+
+const logger = getLogger(__filename);
+
+// Whether we want to mimic the new Promises based RTCPeerConnection or use the
+// old callbacks based one.
+const USE_PROMISES = false;
+
+/**
+ * RTCPeerConnection shim for ORTC based endpoints (such as Edge).
+ *
+ * The interface is based on the W3C specification of 2015, which matches
+ * the implementation of Chrome nowadays:
+ *
+ *   https://www.w3.org/TR/2015/WD-webrtc-20150210/
+ */
+export default class ortcRTCPeerConnection extends yaeti.EventTarget {
+    /**
```

Comment:
Is this for real? Do I need to document standard API? There are *LOT* of methods not documented in lib-jitsi-meet, and they are not even shims of standard APIs.
This is a waste of time. The API is supposed to implement the `RTCPeerConnection` interface defined [here](https://www.w3.org/TR/2015/WD-webrtc-20150210/).